From 2e4ff88545f94693c66e63486255f9bd993d67d7 Mon Sep 17 00:00:00 2001
From: Ionut Nechita <ionut_n2001@yahoo.com>
Date: Thu, 23 Jun 2022 05:18:55 +0000
Subject: [PATCH 1/2] SUNLIGHT: Enable ONDEMAND and CONSERVATIVE gov in default
 options

Description:
 - CPU_FREQ_DEFAULT_GOV_ONDEMAND
 - CPU_FREQ_DEFAULT_GOV_CONSERVATIVE

Bug: N/A
Signed-off-by: Ionut Nechita <ionut_n2001@yahoo.com>
Change-Id: I94d97cd36cf9e20767ffd3ce2f884a0cfbc84f0c
---
 drivers/cpufreq/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/cpufreq/Kconfig b/drivers/cpufreq/Kconfig
index c63de9ed3dc0..e302e12d0f89 100644
--- a/drivers/cpufreq/Kconfig
+++ b/drivers/cpufreq/Kconfig
@@ -79,7 +79,6 @@ config CPU_FREQ_DEFAULT_GOV_USERSPACE
 
 config CPU_FREQ_DEFAULT_GOV_ONDEMAND
 	bool "ondemand"
-	depends on !(X86_INTEL_PSTATE && SMP)
 	select CPU_FREQ_GOV_ONDEMAND
 	select CPU_FREQ_GOV_PERFORMANCE
 	help
@@ -92,7 +91,6 @@ config CPU_FREQ_DEFAULT_GOV_ONDEMAND
 
 config CPU_FREQ_DEFAULT_GOV_CONSERVATIVE
 	bool "conservative"
-	depends on !(X86_INTEL_PSTATE && SMP)
 	select CPU_FREQ_GOV_CONSERVATIVE
 	select CPU_FREQ_GOV_PERFORMANCE
 	help
-- 
2.35.1

